### PR TITLE
feat: preload fonts and reduce icon size

### PR DESCRIPTION
**Pull request by aeneasr**
## Checklist

- [ ] I have read the [contributing guidelines](../blob/master/CONTRIBUTING.md) and signed the CLA.
- [ ] I have referenced an issue containing the design document if my change introduces a new feature.
- [ ] I have read the [security policy](../security/policy).
- [ ] I confirm that this pull request does not address a security vulnerability. 
      If this pull request addresses a security vulnerability, 
      I confirm that I got approval (please contact [security@ory.sh](mailto:security@ory.sh)) from the maintainers to push the changes.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] I have added the necessary documentation within the code base (if appropriate).

### CODE REVIEW

**Review by jonas-jonas — COMMENTED** [review]
image
I don't think this works, unfortunately. Maybe it's just a thing with the preview, though.
image

**Comment by aeneasr** [comment]
Strange, it was working just fine in dev mode. Maybe the plugin loading chain is messed up

**Comment by aeneasr** [comment]
Pagespeed is on fleek now since we don't need to wait, download, parse, swap/render fonts :D

**Comment by aeneasr** [comment]
Yeah, looks like in prod mode docusaurus does not care about my plugin, only in dev mode :(

**Comment by aeneasr** [comment]
It's so messed up - you can't reference the fonts directly in CSS (it always bundles it with content hashes) and you can't change the bundling. So basically you can't pre-load fonts. Webfonts suck...

File: src/plugins/docusaurus-static-fonts/index.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
+const webpack = require("webpack")
+const NodePolyfillPlugin = require("node-polyfill-webpack-plugin")
+
+// eslint-disable-next-line
+module.exports = function (context, options) {
+  return {
+    name: "docusaurus-static-fonts",
+    // eslint-disable-next-line
+    configureWebpack(config, isServer, utils) {
```
Comment (jonas-jonas) [inline src/plugins/docusaurus-static-fonts/index.js:9]: Shouldn't these files just be statically served, as a static asset? Does webpack need to process them?

Comment (aeneasr) [reply src/plugins/docusaurus-static-fonts/index.js:9]: No it doesn't, but I can't figure out how to tell docusaurus to NOT modify them. As soon as CSS uses `url` it bundles the files and ads content hashes. I will probably just reference the file with the content hash in preload as the contents of the file won't likely change anyways.

Comment (jonas-jonas) [reply src/plugins/docusaurus-static-fonts/index.js:9]: Ahh, webpack processes them. Okay I understand.. maybe we can just host these fonts on a static file server or using cloudflare on `ory.sh`

Comment (aeneasr) [reply src/plugins/docusaurus-static-fonts/index.js:9]: We could, but the CSS bundler does its thing and replaces the paths

**Pull request merged**